packages/react-component-library/src/components/Alert/Alert.tsx

```diff
@@ -24,15 +30,17 @@ const VARIANT_ICON_MAP = {
   ),
 }
 
+export type AlertVariant =
```

Comment:
I do not think this is a bad name although it appears to be inconsistent with other naming where we have appended `Type` to the end.